cores/nRF5/sysview/SEGGER/SEGGER_RTT.h

```diff
@@ -83,6 +134,9 @@ Revision: $Rev: 12706 $
 **********************************************************************
 */
 
+#define SEGGER_RTT_NUMBER_OF_TERMINALS 16

```

Comment:
Please do not modify the original RTT file from SEGGER. We should leave it as it is to make it effort-free maintenance